docs/Docker.md

```diff
@@ -0,0 +1,23 @@
+# Installing Docker
+
+## MacOS
+
+### From Docker.com
+
+[Download the dmg](https://www.docker.com/get-started) and install it.
+
+Run Docker and follow through the initial setup dialog (doesn't take long and possibly can be bypassed, but we don't know how).
+
+It won't work until you've completed this step.
+
+### Using Homeboew
+
+```bash
+brew install docker
+```
+
+And then what? Oak's install script fails with
+
+```bash
+Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
+```
```

Comment:
It appears that the docker daemon cannot natively run on Mac (uses Linux kernel features), hence a VM is needed t run it. Here's a guide: https://medium.com/@yutafujii_59175/a-complete-one-by-one-guide-to-install-docker-on-your-mac-os-using-homebrew-e818eb4cfc3

I assume the docker.com setup includes something akin to this.

Reply:
Alternatively one can install the docker.com GUI application via brew through `brew cask install docker`.

Reply:
So simple!